frontend/packages/dev-console/src/components/gitops/details/GitOpsDetails.tsx

```diff
@@ -56,7 +57,21 @@ const GitOpsDetails: React.FC<GitOpsDetailsProps> = ({ envs }) => {
                         )}
                       </StackItem>
                       <StackItem className="co-truncate co-nowrap">
-                        <ResourceLink kind="Project" name={env.environment} />
+                        <span className="co-resource-item">
+                          <ResourceIcon kind="Project" />
+                          {env.cluster ? (
+                            <Link
+                              className="co-resource-item__resource-name"
+                              to={`${env.cluster}/topology/ns/${env.environment}`}
```

Comment:
I think there should be a check to see if the cluster URL is of the current cluster. If yes, then use relative links if not then absolute links. You can check with `route` of `console` deployment in `openshift-console` namespace.

Reply:
I think a normal user will not have access to openshift-console ns :thinking: